eo-maven-plugin/src/test/java/org/eolang/maven/UnphiMojoTest.java

```diff
@@ -283,13 +279,12 @@ void convertsValidXmirAndParsableEO(@Mktmp final Path temp)
     }
 
     @Test
-    @Disabled
     void usesCache(@Mktmp final Path temp) throws Exception {
         new Saved(
             "{⟦std ↦ Φ.org.eolang.io.stdout, y ↦ Φ.org.eolang.x⟧}",
             temp.resolve("target/eo/phi/std.phi")
         ).value();
-        final String hash = "123ZaRiFcHiK321";
+        final String hash = UnphiMojo.FAKE_HASH;
```

Comment:
@vl4ds4m it's totally ok to slightly change the test after you introduce an implementation, let's try to use empty string here and in `UnphiMojo` because it doesn't depend on `tojos`